16) Настроены параметры TS.

Chackmoole opened the pull request:
Типизирован компонент AddUserModal

Chackmoole committed:
16) Настроены параметры TS. Типизирован компонент AddUserModal

Chackmoole committed:
16) TS. Типизирован компонент App

Chackmoole committed:
16) TS. Типизирован компонент DeleteUserModal.

Chackmoole committed:
16) TS. Типизирован компонент EditingUserModal.

Chackmoole committed:
16) TS. Типизированы компоненты Footer & Header(без logo).

Chackmoole committed:
16) TS. Типизированы компоненты TextField & SearchInput

Chackmoole committed:
16) TS. fix компоненты SearchInput

Chackmoole committed:
16) TS. Типизирован компонент MainContent. Удалены логи в SearchInput.

Chackmoole committed:
16) TS. Типизирован компонент UserTable. Удален компонент SearchInput.

Chackmoole committed:
16) TS. Типизирован компонент Button.

Chackmoole committed:
16) TS. Типизирован компонент Filters.

Chackmoole committed:
16) TS. Типизирован компонент Modal.

Chackmoole committed:
16) TS. Типизирован компонент Table.

Chackmoole committed:
16) TS. Типизирован компонент UserForm.

NataliaIlina commented on src/types/index.ts:
```diff
@@ -1,10 +1,16 @@
-export type IUserStatus = 'blocked' | 'active' | 'frozen';
+export type IUserStatus = 'all' | 'blocked' | 'active' | 'frozen';
```
обсудили - в статусе пользователя нет варианта 'all', это наше техническое значение для фильтров

Chackmoole replied:
done

NataliaIlina commented on src/ui/Modal/Modal.tsx:
```diff
@@ -5,12 +5,14 @@ import MuiDialogActions from '@mui/material/DialogActions';
+const Modal = ({ onClose, title, children, onClickAction, open, ...props }: IDialogsProps) => {
   return (
-    <MuiDialog open={open} onClose={onClose}>
+    <MuiDialog open={open} onClose={onClose} {...props}>
```
` {...props}` тут для каких целей?

Chackmoole replied:
done

NataliaIlina commented on src/ui/UserForm/UserForm.tsx:
```diff
@@ -1,9 +1,16 @@
+import { StyledUserForm } from 'ui/UserForm/UserForm.styled';
+
+interface IUserForm {
+  formValues: IUSerFormValue;
```
именование - множественное число и единственное число


уже писала - что этот компонент делает в ui?

Chackmoole replied:
done

NataliaIlina commented on src/components/UsersTable/UsersTable.tsx:
```diff
@@ -1,26 +1,34 @@
+import { Table, TableBody, TableCell, TableHead, TableRow } from 'ui/Table/Table';
+import TextField from 'ui/TextField/TextField';
+
+interface IProps {
+  openModal: () => void;
+  openEditingModal: (user: IUser) => void;
+  openDeleteModal: (user: IUser) => void;
+}
```
openModal => openCreateModal

чтоб прям по красоте

openEditingModal openDeleteModal - выбрать одну форму глагола

Chackmoole replied:
done

NataliaIlina commented:
![image](https://user-images.githubusercontent.com/30752928/203493636-89db720e-f1c0-4c82-94c9-f3955a4c5854.png)

NataliaIlina commented:
из модалок можно удалить это условие, за показ модалок у нас теперь отвечает mui 
![image](https://user-images.githubusercontent.com/30752928/203493761-f14f9ce5-9dd2-4ac8-8614-ff05719fe6e5.png)

Chackmoole committed:
16) TS. Убран лишний статус в типизации IUserStatus
Убрана передача лишних props  в MuiDialog
Исправлено именование IUSerFormValue
UserForm вынесен в components из ui
Переименованы имена state связанные с открытием и закрытием модального окна

Chackmoole committed:
16) В модальных окнах убрано избыточное после введения MUI проверка

Chackmoole committed:
16) Правка в именовании стилизации MainContent.styled.ts

Chackmoole committed:
16) Добавлена поддержка png файлов в проект. Внесены именеия в Header
